### PR TITLE
Storefront purchase redirectUrl

### PR DESCRIPTION
**Pull request by justnero**

### CODE REVIEW

File: openapi/components/schemas/Storefront/Purchase.yaml [hunk 1]
```diff
@@ -69,3 +69,11 @@ properties:
     description: If set would be used to create customer account.
     writeOnly: true
     nullable: true
+  redirectUrl:
+    type: string
+    description: >-
+      The URL where the customer will be redirected after completing the transaction. It may be set to any path at the
```
Comment (ashkarpetin) [inline openapi/components/schemas/Storefront/Purchase.yaml:75]: > It may be set to any path at the related website.

What does it mean?

Comment (justnero) [reply openapi/components/schemas/Storefront/Purchase.yaml:75]: Let's say `Website.url` is `https://example.com` – `redirectUrl` could be set to any of these:

- `https://example.com`
- `https://example.com/some/path`
- `https://example.com/some/path?and=query`
- `https://example.com/some/path?and=query#and-fragment`

Comment (ashkarpetin) [reply openapi/components/schemas/Storefront/Purchase.yaml:75]: Perhaps it should mention that is supports templates:
```
The URL to redirect the end-user when an offsite transaction is completed. Defaults to the website's configured URL. You may use {id} or {result} as placeholders in the URL and we will replace them with the transaction's id and result accordingly.
```

Comment (ashkarpetin) [reply openapi/components/schemas/Storefront/Purchase.yaml:75]: > Let's say `Website.url` is `https://example.com` – `redirectUrl` could be set to any of these:
> 
> * `https://example.com`
> * `https://example.com/some/path`
> * `https://example.com/some/path?and=query`
> * `https://example.com/some/path?and=query#and-fragment`

I think it's clear enough that you provide any url. It's more confusing with it.

Comment (justnero) [reply openapi/components/schemas/Storefront/Purchase.yaml]: Adjusted description. Thank you!

**Review by ashkarpetin — APPROVED** [review]

**Review by mattsanf — APPROVED** [review]

**Pull request merged**